packages/teams-js/src/public/video.ts

```diff
@@ -101,9 +103,9 @@ export namespace video {
   }
 
   /**
-   * video extension should call this to notify Teams Client current selected effect parameter changed.
-   * If it's pre-meeting, Teams client will call videoEffectCallback immediately then use the videoEffect.
-   * in-meeting scenario, we will call videoEffectCallback when apply button clicked.
+   * video extension should call this to notify host client that the current selected effect parameter changed.
```

Comment:
The descriptions in this file could use some wordsmithing, but we'll save that battle for another day.

Reply:
I agree. I think it's okay to prioritize this for later.